### PR TITLE
issue 557 non voting users signup part 2

### LINKED ISSUE #557: Offer non-voting users a way to sign up for the newsletter, on the live homepage and other views
This could take the form of a form underneath the main 'swap my vote' app section with text to the effect of: 

> Not voting this time? 
> Sign up for future news from the Swap My Vote team and Forward Democracy
>[email field] [sign up button]

The newsletter is served by Action Network for which we can provide a form or API details.

### PR DESCRIPTION
**Pull request by baob**
Closes #557

I'll create a review app for this

- extract news signup into a partial (#557)
- add mail sign-up footer to all the different status pages (#557)

### CODE REVIEW

**Comment by baob** [comment]
@tdg  review app created and ready for review.

**Comment by tdg** [comment]
"Not voting this time?" copy should be updated cc #561 


- [ ] Closed warm up 
- [ ] Open 
- [ ] Closed voting 
- [ ] Closed wind down
- [ ] Evergreen

**Comment by tdg** [comment]
closed-wind-down - form shwoing in main div, so 'not voting this time' copy container doesn't make sense

File: app/views/home/_closed_warm_up.html.haml [hunk 1]
```diff
@@ -28,3 +28,6 @@
         %p.text-center
           Make sure you don't miss out -
           = link_to "register to vote now!", "https://gov.uk/register-to-vote"
+
+
+= render partial: "news_sign_up"
```
Comment (aspiers) [inline app/views/home/_closed_warm_up.html.haml]: I don't think we need this one?  This page already has the actionnetwork partial on it.

Comment (aspiers) [reply app/views/home/_closed_warm_up.html.haml]: Fixed

**Review by aspiers — APPROVED** [review]

**Pull request merged**